i18n/de/docusaurus-plugin-content-docs/current/help-center/using-famedly/root-or-jailbreak.mdx

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
Please remove this file from the PR. This article isn't supposed to be in the help center at all.